src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -1721,11 +1749,14 @@ export class SavedObjectsRepository {
       type,
       ...(savedObjectNamespace && { namespace: savedObjectNamespace }),
       ...(savedObjectNamespaces && { namespaces: savedObjectNamespaces }),
-      attributes: normalizedCounterFields.reduce((acc, counterField) => {
-        const { fieldName, incrementBy } = counterField;
-        acc[fieldName] = incrementBy;
-        return acc;
-      }, {} as Record<string, number>),
+      attributes: {
+        ...(upsertAttributes ?? {}),
+        ...normalizedCounterFields.reduce((acc, counterField) => {
+          const { fieldName, incrementBy } = counterField;
+          acc[fieldName] = incrementBy;
+          return acc;
+        }, {} as Record<string, number>),
+      },
```

Comment:
I added the `upsertAttributes` option to `incrementCounter` to be able to create the daily application usage document when it's not present